.eslintrc

```diff
@@ -1,12 +1,7 @@
 {
-  "extends": ["standard"],
+  "extends": ["semistandard"],
```

Comment:
@jflasher ok, Didn't know. I'll fix it and file a PR.
We should coordinate these updates better though.
I've been looking after this project but I see that you also make some changes. There's no need to override each others work.